static/styles/settings.scss

```diff
@@ -1402,11 +1406,11 @@ input[type=checkbox] {
 }
 
 #night-logo-section .realm-logo-block {
-    background-color: hsl(212, 28%, 18%);
+
 }
 
 #day-logo-section .realm-logo-block {
-    background-color: hsla(0, 100%, 100%, 1);
+
```

Comment:
These selectors are now blank and should be removed.

Reply:
ok